Source/PagesController.swift

```diff
@@ -56,7 +81,7 @@ extension PagesController {
         completion: { [unowned self] finished in
           self.pagesDelegate?.pageViewController(self,
             setViewController: viewController,
-            atPage: currentIndex)
+            atPage: self.currentIndex)
```

Comment:
because it is capturing self, making reference cycle (the same as in objc).
To make it visible, you have to specify self.